indico/modules/categories/templates/management/roles.html

```diff
@@ -0,0 +1,23 @@
+{% extends 'categories/management/base.html' %}
+{% from 'categories/management/_roles.html' import render_roles %}
```

Comment:
Didn't we discuss yesterday that we want to reuse the one from events here (and add a new macro param to hide stuff only needed in events, like the email buttons)?